paket.dependencies

```diff
@@ -3,26 +3,26 @@ storage:none
 framework: netstandard2.0
 
 nuget FSharp.Core redirects:force
-nuget Fable.Core
+nuget Fable.Core prerelease
 nuget Fable.Elmish
 nuget Fable.Elmish.React
 nuget Fable.Elmish.Browser
 nuget Fable.Elmish.Debugger
 nuget Fable.Elmish.HMR
-nuget Fable.PowerPack
-nuget Fable.React
+nuget Fable.PowerPack prerelease
+nuget Fable.React prerelease
 nuget Fulma prerelease
 nuget Fulma.Extensions prerelease
 nuget Fulma.Elmish prerelease
-nuget Thoth.Json
+nuget Thoth.Json prerelease
 
 group netcorebuild
     storage:none
     source https://www.nuget.org/api/v2
     framework: netstandard2.0
 
     clitool dotnet-fake
-    clitool dotnet-fable
+    clitool dotnet-fable 2.0.0-alpha-028
```

Comment:
Can't we use `prerelease` here ? 
So user can update easily there deps and also I think now we have beta versions